components/atoms/icon/icon/props.js

```diff
@@ -31,5 +31,5 @@ export const IconPropTypes = {
 
 export const IconDefaultProps = {
   fixedWidth: true,
-  prefix: 'far'
+  prefix: 'fas'
```

Comment:
@Ali-sharafi-git @mhsnprvr3 Great so we have the default - so we dont need to default in other places :)